freedomofpress/securedrop
# Use the correct description for the App Server SSH Onion URL entry in the KeePassXC Database

## Status

Ready for review

## Description of Changes

Fixes #6523 

Changes proposed in this pull request:

The description for "App Server SSH Onion URL" was originally set to:  `Credentials for the App and Monitor Server, set during the Ubuntu install and required to be the same.`

I updated the description to be: `The Onion Service values required to connect to the App Server.`

Which also aligns with the description for "Monitor Server SSH Onion URL" for the sake of consistency.

## Testing

How should the reviewer test this PR?

After checking out the branch, open the `tails_files/securedrop-keepassx.kdbx` with the KeePassXC application. Unlock the database by using an empty password, then verify the details are set correctly for the "App Server SSH Onion URL"  entry.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

This is only used as a template during setup of a SecureDrop instance. Since this is only a template and not used directly by the SecureDrop application, I don't foresee any issues once deployed.

## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you added or removed a file deployed with the application:


### If you made non-trivial code changes:


Choose one of the following:
